fulp_modules/lisa/velvet_fu.dm

```diff
@@ -0,0 +1,318 @@
+	return TRUE
+
+/// Full Thrust
+/datum/martial_art/velvetfu/proc/fullThrust(mob/living/A, mob/living/D)
+	log_combat(A, D, "full thrust (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] thrusted into [D]!</span>", \
+					"<span class='userdanger'>You're thrusted by [A]!</span>", \
+					"<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You quickly and fashionably thrust into [D]!</span>")
+	A.adjustStaminaLoss(60)
+	if(prob(70) && !D.stat)
+		D.Knockdown(4 SECONDS) /// Perfect for Iron Hooving
+	D.apply_damage(10, A.get_attack_type(), BODY_ZONE_CHEST, wound_bonus = CANT_WOUND)
+	playsound(get_turf(A), 'sound/weapons/cqchit1.ogg', 50, TRUE, -1)
+	return TRUE
+
+/// Minor Iris - The only move guaranteed to have all its effects.
+/datum/martial_art/velvetfu/proc/minorIris(mob/living/A, mob/living/D)
+	log_combat(A, D, "minor iris (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] slashes [D] rapidly and repeatedly!</span>", \
+					"<span class='userdanger'>You're slashed several times by [A]!</span>", \
+					"<span class='hear'>You hear several sickening sounds of flesh slashing flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You swiftly and repeatedly slash at [D], truly a master attack!</span>")
+	A.adjustStaminaLoss(80)
+	var/obj/item/bodypart/limb = D.get_bodypart(ran_zone(A.zone_selected))
+	var/datum/wound/slash/moderate/crit_wound = new
+	crit_wound.apply_wound(limb)
+	D.apply_damage(30, A.get_attack_type(), wound_bonus = CANT_WOUND) /// Don't wound here.
+	playsound(get_turf(A), 'sound/weapons/bladeslice.ogg', 50, TRUE, -1)
+	return TRUE
+
+
+/*
+ *	Intents:
+ *
+ *	Intents are meant to be what the Combo Dial is in Lisa the Pointless
+ *	Grabbing and Disarming both aren't possible. These are replaced with different versions of Harm.
+ */
+
+/datum/martial_art/velvetfu/disarm_act(mob/living/A, mob/living/D)
+	if(HAS_TRAIT(A, TRAIT_PACIFISM))
+		return FALSE
+	var/datum/dna/dna = A.has_dna()
+	if(dna?.check_mutation(HULK))
+		return FALSE
+	add_to_streak("D",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "disarmed (Velvet-Fu)")
+	A.do_attack_animation(D)
+	var/picked_hit_type = "side kick"
+	var/bonus_damage = 10 // Kicking deals more damage
+	if(D.body_position == LYING_DOWN)
+		bonus_damage += 8
+		picked_hit_type = "iron hooved"
+	D.apply_damage(bonus_damage, A.get_attack_type())
+	D.visible_message("<span class='danger'>[A] [picked_hit_type]ed [D]!</span>", \
+					"<span class='userdanger'>You're [picked_hit_type]ed by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You [picked_hit_type] [D]!</span>")
+	playsound(D, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	return TRUE
+
+/datum/martial_art/velvetfu/grab_act(mob/living/A, mob/living/D)
```

Comment:
does this still have the second long click cooldown of normal grab_act()s?

Reply:
It should, yes